tests/PolykeyClient.test.ts

```diff
@@ -77,11 +81,16 @@ describe('PolykeyClient', () => {
     expect(await session.readToken()).toBeDefined();
     await pkClient.destroy();
     expect(await session.readToken()).toBeUndefined();
-    await webSocketClient.destroy({ force: true });
   });
   test('end to end with authentication logic', async () => {
-    const webSocketClient = await WebSocketClient.createWebSocketClient({
-      config: {
+    const options: PolykeyClientOptions = {
+      nodePath: nodePath,
+      tokenBase: config.paths.tokenBase,
+      rpc: {
+        callTimeoutTime: config.defaultsSystem.rpcCallTimeoutTime,
+        parserBufferSize: config.defaultsSystem.rpcParserBufferSize,
+      },
+      wsConfig: {
         verifyPeer: true,
         verifyCallback: async (certs) => {
           await clientUtils.verifyServerCertificateChain(
```

Comment:
The custom verification options should be set inside the `PolykeyClient`.

Reply:
It is now done, we are taking `nodeId` inside `PolykeyClient`.